frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1075,15 +1087,32 @@ const ManageHostsPage = ({
     setIsUpdatingHosts(true);
 
     const teamId = isAnyTeamSelected ? currentTeamId ?? null : null;
-    const labelId = selectedLabel?.id;
 
     try {
       await (isAllMatchingHostsSelected
         ? hostsAPI.destroyByFilter({
```

Comment:
Does it make sense to put all of these filters into a `filters` variable and then use that for both the above `action` and here, instead of listing them out in both places?